db/migrate/20150921204114_add_vmware_ro_datastores_to_hosts_storages.rb

```diff
@@ -0,0 +1,6 @@
+class AddVmwareRoDatastoresToHostsStorages < ActiveRecord::Migration
+  def change
+    add_column :hosts_storages, :read_only, :boolean
+    add_column :hosts_storages, :id, :primary_key
```

Comment:
This may not default to a `bigint`, nor will it acquire the region number sequencing because that is defined in `create_table` [here](https://github.com/ManageIQ/manageiq/blob/master/lib/extensions/ar_migration.rb#L3).  You will probably have to change this either here or add a method to ar_migration.

@tenderlove Do you have any thoughts on this?  I'm trying to think of a way to make this less error prone.  Would be nice if the code that is in this PR *just worked*.  I guess we could patch `add_column` in ar_migration.rb as well?